ObinsKit.ObinsKit version 1.1.5

headblockhead opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17267)

headblockhead committed:
ObinsKit.ObinsKit version 1.1.5

OfficialEsco commented on manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml:
```diff
@@ -0,0 +1,11 @@
+PackageIdentifier: ObinsKit.ObinsKit
+PackageVersion: 1.1.5
+PackageLocale: en-US
+Publisher: Taicang Zhigengniao Information Technology Co. Ltd
```
```suggestion
Publisher: Taicang Zhigengniao Information Technology Co.,Ltd
```
Needs to match ARP

headblockhead replied:
Done!

OfficialEsco commented on manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml:
```diff
@@ -0,0 +1,11 @@
+Publisher: Taicang Zhigengniao Information Technology Co. Ltd
+PackageName: ObinsKit
+License: Copyright © 2019 Taicang Zhigengniao Information Technology Co.,Ltd
+ShortDescription: ObinsKit installer
```
Can you add a better description?

headblockhead replied:
Done!

headblockhead committed:
Added missing comma in publisher name

Taicang Zhigengniao Information Technology Co. Ltd ->
Taicang Zhigengniao Information Technology Co.,Ltd

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

headblockhead committed:
Added Better Description

Obinskit Installer -> AnnePro keyboard settings manager.

OfficialEsco commented:
Close and re open the PR since the bot crashed
